README.md

```diff
@@ -64,8 +79,9 @@ And wait for a few days. Your checkpoints will be dumped in output_networks/cele
 For celebaHQ:
 
 ```
-python datasets.py celebaHQ $PATH_TO_CELEBAHQ -o $OUTPUT_DATASET - f
-python train.py PGAN -c config_celebaHQ.json --restart -n celebaHQ
+python datasets.py celebaHQ $PATH_TO_CELEBAHQ -o $OUTPUT_DATASET - f  # Prepare the dataset and build the configuration file.
```

Comment:
not sure of this << - f >> ?

Reply:
-f accelerate the training by saving intermediate images of smaller sizes.